src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -253,6 +254,40 @@ def _validate_output(self):
 
             self.logger.critical(self.name, ErrorCode.INVALID_OUTPUT, error_msg)
 
+    def _checksum_output_products(self):
+        """
+        Generates a dictionary mapping output product file names to the
+        corresponding MD5 checksum digest of the file's contents.
+
+        The output products to generate checksums for is determined by scanning
```

Comment:
Suggested re-wording:  Determines the output products from which to generate checksums  by scanning the output product location ....